pygyro/advection/accelerated_advection_steps.py

```diff
@@ -223,95 +234,96 @@ def poloidal_advection_step_impl( f, dt, v, rPts, qPts,
         for j in range(nPts_r):
             # Step one of Heun method
             # x' = x^n + f(x^n)
-            drPhi_0[i,j]/=rPts[j]
-            dthetaPhi_0[i,j]/=rPts[j]
-            endPts_k1_q[i,j] = qPts[i] - drPhi_0[i,j]*multFactor
-            endPts_k1_r[i,j] = rPts[j] + dthetaPhi_0[i,j]*multFactor
+            drPhi_0[i, j] /= rPts[j]
+            dthetaPhi_0[i, j] /= rPts[j]
+            endPts_k1_q[i, j] = qPts[i] - drPhi_0[i, j]*multFactor
+            endPts_k1_r[i, j] = rPts[j] + dthetaPhi_0[i, j]*multFactor
 
     multFactor *= 0.5
 
-    norm=tol+1
-    while (norm>tol):
-        norm=0.0
+    norm = tol+1
+    while (norm > tol):
+        norm = 0.0
         for i in range(nPts_q):
             for j in range(nPts_r):
                 # Handle theta boundary conditions
-                while (endPts_k1_q[i,j]<0):
-                    endPts_k1_q[i,j]+=2*pi
-                while (endPts_k1_q[i,j]>2*pi):
-                    endPts_k1_q[i,j]-=2*pi
+                while (endPts_k1_q[i, j] < 0):
+                    endPts_k1_q[i, j] += 2*pi
+                while (endPts_k1_q[i, j] > 2*pi):
+                    endPts_k1_q[i, j] -= 2*pi
 
```

Comment:
These can be replaced with modulo operators

Reply:
yes